src/javascript/JContent/PageComposerRoute/EditFrame/Box.scss

```diff
@@ -1,3 +1,6 @@
+$color-accent_light: rgba(99, 209, 255, 0.6);
+$color-accent: rgb(0, 160, 227);
```

Comment:
I don't know why the moonstone CSS properties don't work here.
It seems the others moonstone variables used in this file are associated with unused CSS.

Reply:
The moonstone variables are not copied into the iframe dom, so the css which are used there cannot use them